getAllFeatureFlags/go.mod

```diff
@@ -0,0 +1,14 @@
+module getAllFeatureFlags
```

Comment:
The entry for this function is missing in `template.yaml` file

Reply:
done